.github/workflows/winget.yml

```diff
@@ -0,0 +1,13 @@
+name: Publish to WinGet
+on:
+  release:
+    types: [released]
```

Comment:
It usually takes a while for binaries to be built after releases. Will this work correctly? Or does it need to wait for other workflows to finish first?

Reply:
The will retry every 10 minutes, until 1 hour, chance, it won't fail.